packages/client/hmi-client/src/composables/useChartSettings.ts

```diff
@@ -85,16 +83,29 @@ export function useChartSettings(
 		});
 	};
 
-	const addComparisonChartSettings = () => {
+	const addEmptyComparisonChart = () => {
```

Comment:
Why not just using existing `addMultiVariableChartSetting` with empty variable array?